sycl/test-e2e/RawKernelArg/arg_struct.cpp

```diff
@@ -0,0 +1,77 @@
+// REQUIRES: aspect-usm_shared_allocations
+// REQUIRES: ocloc && level_zero
+
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+// Tests raw_kernel_arg which is used to pass user-defined data types
+// (structures) as kernel arguments.
+#include <sycl/detail/core.hpp>
+#include <sycl/usm.hpp>
+
+constexpr size_t NumArgs = 4;
```

Comment:
Should this be `2`?

Reply:
Right, copy-paste side-effect. Addressed in b59cfeb31a17e57716d8a5f53ccc8e61cc417146